src/main/java/gift/Product.java

```diff
@@ -0,0 +1,58 @@
+package gift;
+
+public class Product {
+    Long id;
```

Comment:
private과 같은 적절한 접근제어자를 사용하면 접근을 제한할 수 있습니다ㅎㅎ
자바가 제공하는 접근제어자에 대해서도 알아보시면 좋을 것 같아요!
```suggestion
    private Long id;
```